XenAdmin/TabPages/SnapshotsPage.resx

```diff
@@ -1032,11 +1032,130 @@
   <data name="snapshotTreeView.Font" type="System.Drawing.Font, System.Drawing">
     <value>Microsoft Sans Serif, 8.25pt, style=Bold</value>
   </data>
+  <data name="snapshotTreeView.Items" mimetype="application/x-microsoft.net.object.binary.base64">

```

Comment:
In general, could all developers please look through their own diffs before submitting the pull request? Some things could be caught before a reviewer has to get involved. Thanks.